packages/helix_editor.rb

```diff
@@ -0,0 +1,53 @@
+require 'package'
+
+class Helix_editor < Package                 # The first character of the class name must be upper case
+  description 'A vim inspired editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/refs/tags/22.12.tar.gz'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972'   # Use the command "sha256sum"
+  
+  depends_on 'rust' => :build
+
+  @no_fhs = true
+  
+  def self.build
+   puts "Building. This may be long."
+   system "cargo build \
+      --release \
+      --locked \
+     "
+  end
+
+  def self.install
+    # Copy executable
+    helix_executable_dest_dir = "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.mkdir_p("#{helix_executable_dest_dir}")
+    FileUtils.cp("./target/release/hx", "#{helix_executable_dest_dir}")
```

Comment:
It might be better to use `FileUtils.install` to ensure the binary is executable.  Something like:
```
FileUtils.install './target/release/hx', helix_executable_dest_dir, mode: 0o755
```

Reply:
done